docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -96,7 +105,7 @@ export API_KEY="sk-..."
 
 Once you have your Langflow server URL, try calling these endpoints that return Langflow metadata.
 
-If authentication is required, include an `x-api-key` header or query parameter with a valid [Langflow API key](/configuration-api-keys), such as `-H 'x-api-key: $API_KEY'`.
+If authentication is required, include an `x-api-key` header or query parameter with a valid [Langflow API key](/configuration-api-keys), such as `-H "x-api-key: $API_KEY"`.
 
```

Comment:
Please add the auth header to all 3 endpoints here: https://d5rxiv0do0q3v.cloudfront.net/langflow-drafts/docs-add-api-auth-requirement/api-reference-api-examples#try-some-langflow-api-requests (Get version, Get config, Get all)